src/cmds/get/events.ts

```diff
@@ -0,0 +1,55 @@
+import { log, logFailure, logWarning } from '../../logger'
+import chalk from 'chalk'
+import { client } from '../../api/client'
+import { isValidContractGroup } from '../../utils/validators'
+import { StringKeyMap } from '../../types'
+
+const CMD = 'events'
+
+function addGroupEventsCmd(cmd) {
+    cmd.command(CMD)
+        .requiredOption('--group <group>', 'Contract group to get events for')
```

Comment:
This is the first command I thought to myself "a shorthand flag would be nice".  Let's go ahead and add that in:
```typescript
.requiredOption('-g, --group <group>', ...
```